tests/baselines/reference/ambientShorthandFindAllRefs.baseline.jsonc

```diff
@@ -106,7 +106,7 @@
      },
      {
       "text": "\"jquery\"",
-      "kind": "stringLiteral"
```

Comment:
I think this means that `writeSymbol` call added into the emitter might change how we colorize this quickinfo because of the now attached symbol data in VS (in vscode the quickinfo colorization isn't based on these kinds anymore). Specifically, in the default theme, I think the string's going to be a blue color instead of string color with this? Which seems maybe subjective but less good? I dunno if it matters, and don't actually know if that's still true for VS.